DEVELOPMENT.md

```diff
@@ -63,27 +64,6 @@ Before you start working you will need to install the dependencies for the clien
 1. Move into the `client` directory (`cd client`)
 2. Run `npm install`
 
-### Enable ESLint in VS Code
```

Comment:
I would be in favor of leaving this in but maybe changing the wording to be more like "just in case it hasn't been enabled yet"

Reply:
I thought about that, but there's been some confusion since we start including that in Lab 3, so they've seen it a ton by now and there was some confusion in Lab 4 about whether they'd already done it or not.

Maybe this should move into the `CODE_QUALITY_CHECKS.md` file?